File: smp2017-Python-Abhinav/CS50-Solutions/pset1/mario.c
#include<stdio.h>
#include<cs50.h>

int main()
{
    printf("Enter Height");
    int a = GetInt();
    int f=0;
    int b=0;
    while(a>23 || a<0)
    {
        printf("Enter Height");
        a = GetInt();
    }
    for(f=1;f<=a;f++)
    {
        b=a-f;
        for(int z=0;z<b;z++)
        {
            printf(" ");
        }
        for(int i=0;i<f;i++)
        {
            printf("#");
        }
        printf("  ");
        for(int i=0;i<f;i++)
        {
            printf("#");
        }
        printf("\n"); 
    }

}

File: smp2017-python-kevin/cs50/pset6/mario.py
import cs50

print("Enter no: not bigger than 23: ", end="")
n=cs50.get_int()
        
for i in range(1,n+1):
    for j in range(n-i,0,-1):
        print(" ", end="")
    for k in range(1,i+1):
        print("#", end="")
    print("  ",end="")
    for l in range(1,i+1):
        print("#",end="")
    print("\n",end="")